package.json

```diff
@@ -22,12 +22,12 @@
   "sideEffects": false,
   "scripts": {
     "test": "xo && nyc ava",
-    "prepublishOnly": "rm -rf dist && yarn run build",
+    "prepublishOnly": "rimraf dist && yarn run build",
     "dev": "start-storybook -p 9009",
     "build": "tsc",
     "build-storybook": "build-storybook -s .storybook/static -o .out",
     "release": "np --no-publish",
-    "benchmark": "echo ui-box && react-benchmark tools/benchmarks/box.js",
+    "benchmark": "echo ui-box && react-benchmark tools/benchmarks/box.ts",
```

Comment:
Yeah, the benchmark tool doesn't know how to resolve the `.tsx?` files. Not sure if we need to upgrade `react-benchmark` or what, but we could always keep the benchmark files strictly `.js` and/or point `box.ts` to consume files from the build instead of the source.

Reply:
Playing around with it a bit, I don't think just keeping it `.js` will work either since the source is typescript. If I could point it to `../dist` instead of `../src` that'd solve things but I'm unsure if you'd like that? 

cc @Rowno maybe you can give some additional insights since you created react-benchmark? 🙏

Reply:
Apparently there's a minor parsing glitch in react-benchmark due to its webpack config. I put up a little fix for it over at @Rowno's repo and if that's published I can also add my changes to the benchmarking here (pointing a .js file to ../dist)